### PR TITLE
Increment core-amqp version after 1.6.0 release

### PR DESCRIPTION
**Pull request by YijunXieMS**

### CODE REVIEW

**Review by samvaity — APPROVED** [review]

File: sdk/servicebus/azure-messaging-servicebus/pom.xml [hunk 1]
```diff
@@ -47,7 +47,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-amqp</artifactId>
-      <version>1.6.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure:azure-core-amqp;dependency} -->
+      <version>1.7.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-amqp;current} -->
```
Comment (samvaity) [inline sdk/servicebus/azure-messaging-servicebus/pom.xml:50]: This should take unreleased_com depedency of core-amqp if SB needs active changes from the core-amqp source.

File: eng/versioning/version_client.txt [hunk 2]
```diff
@@ -13,7 +13,7 @@ com.azure:azure-communication-common;1.0.0-beta.2;1.0.0-beta.3
 com.azure:azure-communication-administration;1.0.0-beta.2;1.0.0-beta.3
 com.azure:azure-communication-sms;1.0.0-beta.2;1.0.0-beta.3
 com.azure:azure-core;1.9.0;1.10.0-beta.1
-com.azure:azure-core-amqp;1.6.0;1.6.0-beta.1
+com.azure:azure-core-amqp;1.6.0;1.7.0-beta.1
```
Comment (samvaity) [inline eng/versioning/version_client.txt:16]: Update L141
unreleased_com.azure:azure-core-amqp;1.6.0-beta.1 ==> 
unreleased_com.azure:azure-core-amqp;1.7.0-beta.1

File: sdk/servicebus/azure-messaging-servicebus/pom.xml (hunk 1 above)
Comment (YijunXieMS) [reply sdk/servicebus/azure-messaging-servicebus/pom.xml]: Corrected to use unreleased_

File: eng/versioning/version_client.txt (hunk 2 above)
Comment (YijunXieMS) [reply eng/versioning/version_client.txt:16]: Updated

**Pull request merged**